app-information.html

```diff
@@ -0,0 +1,369 @@
+          intended to be used for distributing the web app. The
+          <a>iarc_rating_id</a> member only takes a single certification code.
+          The same code can be shared across participating storefronts, as long
+          as the distributed product remains the same (i.e., doesn’t serve
+          totally different code paths depending on user agent sniffing and the
+          like) and the other storefronts support it.
+        </p>
+        <div class="informative">
+          <p>
+            The following shows a very simple <a>manifest</a> with the
+            <code>iarc_rating_id</code> member.
+          </p>
+          <pre class="example json" title="very simple manifest">
+          {
+            "name": "Donate App",
+            "description": "This app helps you donate to worthy causes.",
+            "iarc_rating_id": "e84b072d-71b3-4d3e-86ae-31a8ce4e53b7",
+            "icons": [{
+              "src": "images/icon.png",
+              "sizes": "192x192"
+            }]
+          }
+        </pre>
+        </div>
+        <p>
+          More information on the IARC can be found at: <a href=
+          "https://www.globalratings.com/how-iarc-works.aspx">How IARC
+          Works</a> and <a href=
+          "https://www.globalratings.com/for-developers.aspx">How developers
+          can get their games and apps rated with IARC</a>.
+        </p>
+      </section>
+      <section>
+        <h3>
+          <code title="">screenshots</code> member
+        </h3>
+        <p>
+          The <dfn>screenshots</dfn> member is an <a>array</a> of
+          {{ManifestImageResource}}s, representing the web application in
+          common usage scenarios.
+        </p>
+        <p>
+          For accessibility purposes, authors MUST provide a
+          {{ImageResource/label}} for each screenshot.
+        </p>
+      </section>
+    </section>
+    <section class="appendix">
+      <h2>
+        Acknowledgements
+      </h2>
+      <p>
+        TBD
+      </p>
+    </section>
+    <section id="conformance" data-status="done" class="appendix">
+      <p>
+        There is only one class of product that can claim conformance to this
+        specification: a <dfn>user agent</dfn>.
+      </p>
+      <p class="note">
+        Although this specification is primarily targeted at web browsers, it
+        is feasible that other software could also implement this specification
+        in a conforming manner. For instance, search engines, or crawlers,
+        could find and process manifests to build up catalogs of sites that
+        potentially work as installable web applications.
+      </p>
+    </section>
+    <section id="internationalization" class="appendix informative">
+      <h2>
+        Internationalization
+      </h2>
+      <p>
+        It is expected that authors will localize the content of all text
+        strings defined in this document based on
+        {{WebAppManifest/internationalization}}.
```

Comment:
This is not a thing :)

Reply:
Is there a Respec-friendly means of linking to that section in the spec?

Reply:
I think I found one. Is this it?

>  <a data-cite="MANIFEST#internationalization">…</a>

Reply:
yeah, we could go with that... alternatively, we should export something from Web Manifest.